cli-process-instances.js

```diff
@@ -0,0 +1,36 @@
+// Configure environment variables based on .env file
+import "dotenv/config";
+
+import processinstances from "./camunda-process-instances.js";
```

Comment:
When we do import this into the original cli.js, I think it should import using camel case (processInstances) for readability.